<ui version="4.0" >
 <class>DialogTags</class>
 <widget class="QDialog" name="DialogTags" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listTagsServeur" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listTagsLocal" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>10</y>
     <width>241</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Buttoncopiertag" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>110</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>>></string>
   </property>
  </widget>
  <widget class="QPushButton" name="Buttonsupprimertag" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>180</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTags" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>320</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonAjoutTag" >
   <property name="geometry" >
    <rect>
     <x>530</x>
     <y>320</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>+</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
